### PR TITLE
Added a file in Task1

### PR DESCRIPTION
**Pull request by AnkitaxPriya**

### CODE REVIEW

File: Task 3/Knapsack/knapsack.cpp [hunk 1]
```diff
@@ -0,0 +1,55 @@
+#include <iostream>
```
Comment (nimishbongale) [inline Task 3/Knapsack/knapsack.cpp:1]: Can we have a small explanation & sample Input/Output cases as a part of a multiline comment at the top here?

**Review by nimishbongale — COMMENTED** [review]
Great work @AnkitaxPriya! 😄 

It's nearly perfect! Just one small change suggestion.

Let's get this merged ASAP!

File: Task 3/Knapsack/knapsack.cpp (hunk 1 above)
Comment (AnkitaxPriya) [reply Task 3/Knapsack/knapsack.cpp:1]: Could you please add a link to this question? 
It explains the question along with the example. 
https://leetcode.com/problems/sort-colors/

Comment (nimishbongale) [reply Task 3/Knapsack/knapsack.cpp:1]: Alright @AnkitaxPriya.

Go ahead and add that link as a part of a comment on the top of the program!

**Review by nimishbongale — APPROVED** [review]
Awesome work @AnkitaxPriya! 🚀 

There was a merge conflict, but I resolved it.

You are now ready to contribute to cool open source projects!

**Pull request merged**